SteamKit2/Tests/WebAPIFacts.cs

```diff
@@ -11,7 +11,15 @@ public void WebAPIHasDefaultTimeout()
         {
             var iface = WebAPI.GetInterface( new Uri("https://whatever/"), "ISteamWhatever" );
 
-            Assert.Equal( iface.Timeout, 1000 * 100 );
+            Assert.Equal( iface.Timeout, TimeSpan.FromSeconds( 100 ) );
+        }
+
+        [Fact]
+        public void WebAPIAsyncHasDefaultTimeout()
+        {
+            var iface = WebAPI.GetAsyncInterface( new Uri("https://whatever/"), "ISteamWhatever" );
+
+            Assert.Equal( iface.Timeout, TimeSpan.FromSeconds( 100 ) );
```

Comment:
Maybe we should make it less hardcoded, such as `internal const int defaultTimeout = 100` in `WebAPI`? Just an idea though. If we ever want to change timeout it'd be better for tests to match it too, unless we intentionally don't want to change default timeout.

Reply:
Personal opinion only, unit testing should not depend upon the same constants when that's the thing it's testing. If you want to change tested functionality, you should be required to be make at minimum two changes - one to functional code and one to test code.

Reply:
This is fine, it's a matter of whether we want to test that our default timeout is set, or whether we want to test that and in addition to that test it also for specific hardcoded value. It's a matter of what really test should be doing, in your case I assume second, which is totally OK.